src/main/controllers/PastEmployerController.ts

```diff
@@ -23,17 +25,26 @@ export default class PastEmployerController {
     saveForLater: saveForLaterButton,
   };
 
-  constructor() {
+  constructor(private logger: LoggerInstance) {
     this.form = new Form(<FormFields>this.pastEmployerFormContent.fields);
   }
 
   public post = (req: AppRequest, res: Response): void => {
+    const session = req.session;
     const redirectUrl = conditionalRedirect(req, this.form.getFormFields(), YesOrNo.YES)
       ? PageUrls.STILL_WORKING
       : PageUrls.HOME;
```

Comment:
Please update the redirect url so it navigates to the respondent name page then remove the TODO

Reply:
fixed